### ISSUE TITLE
circle list all failed crashing hubot

### ISSUE THREAD

**Issue by Linuturk** [hubot-scripts/hubot-circleci#11]
```
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Received error [object Object]
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR undefined
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Exiting in 1 second
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Received error [object Object]
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR undefined
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Exiting in 1 second
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Received error [object Object]
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR undefined
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Exiting in 1 second
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Received error [object Object]
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR undefined
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Exiting in 1 second
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Received error [object Object]
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR undefined
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Exiting in 1 second
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Received error [object Object]
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR undefined
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] ERROR Exiting in 1 second
Jan 21 19:01:47 xxxx sh[21414]: [Wed Jan 21 2015 19:01:47 GMT+0000 (UTC)] INFO Slack client closed
```

**Comment by Linuturk** [comment]
Working on diagnosing and solving this.

**Comment by Linuturk** [comment]
This seems to only happen when a large number of entries are returned by the API. Could be related to rate limiting or pagination.

**Comment by Linuturk** [comment]
https://circleci.com/docs/api#projects << pagination not supported on the `projects` endpoint.

**Comment by dylanlingelbach** [comment]
Fixed in #12. 

Published in 0.8.1
